### PR TITLE
First draft of Norwegian translation

### PR DESCRIPTION
**Pull request by repperille**
## Purpose

This PR holds the first draft of the Norwegian translation of error messages

## Context

Fixes: #912

## Changes

...

## How to test this PR

...

### CODE REVIEW

File: share/nb.po [hunk 1]
```diff
@@ -2,12 +2,73 @@ msgid ""
...
 
+msgid "Domain name required"
+msgstr "Domenenavn påkrevd"
+
+msgid ""
+"The domain name is not a valid IDNA string and cannot be converted to an A-"
+"label"
+msgstr ""
+"Domenenavnet er ikke en gyldig IDNA streng og kan ikke konverteres til en "
+"A-post"
```
Comment (matsduf) [inline share/nb.po:22]: Is "A-post" really correct here? Below you have used "label" for "label"? In the translation you have "post" for "record".

File: share/nb.po [hunk 2]
```diff
@@ -2,12 +2,73 @@ msgid ""
...
+msgstr "Domenenavn eller label er for lang"
+
+msgid "Unknown profile"
+msgstr "Ukjent profil"
+
+msgid "Invalid IP address"
+msgstr "Ugylding IP-adresse"
+
+msgid "Unkown language string"
+msgstr "Ukjent pråk"
```
Comment (matsduf) [inline share/nb.po:40]: "språk-tag" as below?

**Review by matsduf — CHANGES_REQUESTED** [review]
Does "first draft" mean that you plan to do more work on it?

File: share/nb.po (hunk 1 above)
Comment (repperille) [reply share/nb.po:22]: My bad, embarrassed.

File: share/nb.po (hunk 2 above)
Comment (repperille) [reply share/nb.po:40]: Well spotted!

**Review by matsduf — APPROVED** [review]
Looks good. I approve.

**Review by mattias-p — APPROVED** [review]

**Pull request merged**